src/components/PrimaryNavItems.tsx

```diff
@@ -1,17 +1,25 @@
 import Link from 'next/link';
 import { getNavigationItems } from '../utils/getNavigationItems';
 
+import { useState } from 'react';
+
 const navigationItems = getNavigationItems({
   scope: 'PRIMARY',
 });
 
 export const PrimaryNavItems = () => {
+  const [active, setActive] = useState(0);
   return (
     <>
-      {navigationItems.map((navigationItem) => (
+      {navigationItems.map((navigationItem, index) => (
         <li key={navigationItem.title}>
           <Link href={navigationItem.href!}>
-            <span className="hover:text-sky-500 dark:hover:text-sky-400">
+            <span
+              onClick={() => setActive(index)}
+              className={`${
+                active === index ? 'text-sky-500' : 'none'
```

Comment:
remove none class 
` className={`${active === index && 'text-sky-500'}  hover:text-sky-500 dark:hover:text-sky-400`}`
OR
`className={clsx(active === index && 'text-sky-500','hover:text-sky-500 dark:hover:text-sky-400' )}`

Reply:
Use `clsx` for conditional className.

Reply:
Done